### ISSUE TITLE
[Feature Request] add i18n API to ngx-auth-firebaseui-user

### ISSUE THREAD

**Issue by geromegrignon** [AnthonyNahas/ngx-auth-firebaseui#608]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [x] feature request
```

#### What would like to see implemented?
The `ngx-auth-firebaseui-user` component doesn't the same API as others components for i18n support.

I plan to make a PR to address tihs issue.
The point of this issue is to track the missing feature for `ngx-auth-firebaseui-user` and to let people know a PR is in progress.

### Mention any other details that might be useful

I'll address a request to the Angular community  to review the added translated texts.

**Comment by AnthonyNahas** [comment]
thnak you

**Comment by andi-di** [comment]
+1
We use the ngx-auth-firebase-user component and we miss these inputs as well. 
It is not possible to translate the labels like in the other components.
It would be nice if you could add this functionality.

**Comment by geromegrignon** [comment]
@AnthonyNahas if you don't mind, i'll submit the PR in February.
I plan to use it as a way to show students how to contribute to an open source project during a live coding session.

**Comment by geromegrignon** [comment]
closed by #611
